src/ripple/core/impl/Config.cpp

```diff
@@ -411,26 +413,22 @@ void Config::loadFromString (std::string const& fileContents)
     if (getSingleSection (secConfig, SECTION_SSL_VERIFY, strTemp, j_))
         SSL_VERIFY          = beast::lexicalCastThrow <bool> (strTemp);
 
-    if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, strTemp, j_))
+    if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, VALIDATION_SEED, j_))
```

Comment:
I don't think this is necessary (or this comment is outdated). `getSingleSection` does not do anything with the `strValue` parameter unless it is going to return `true`.